bumble/pandora/l2cap.py

```diff
@@ -234,18 +240,18 @@ async def Connect(
         if not l2cap_channel:
             return ConnectResponse(error=COMMAND_NOT_UNDERSTOOD)
 
-        channel = self.channel_to_proto(l2cap_channel)
-        return ConnectResponse(channel=channel)
+        pandora_channel = self.craft_pandora_channel(connection_handle, l2cap_channel)
```

Comment:
Should you save the channel in `onServiceConnected` here as well ?

Reply:
The `connect` l2cap function is awaiting for the connection to happen. However the pandora_channel was not registered internally indeed. Just added the missing piece of code.